angular/README.md

```diff
@@ -1,10 +1,25 @@
-# MyApp
+# WxCC Widget Starter - Angular
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.2.11.
 
-## Development server
+This widget starter can be run on `localhost` to be reviewed. In case you (the developer) are using this widget starter as a foundation of your custom widget, running the sandbox environment on `localhost` would be a perfect way to build your widget's logic and interface.
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+**Note**: please keep in mind that we recommend to design every widget as responsive. With the exception of widgets for Agent or Supervisor Desktop **header** zone (where widgets should not exceed 64px height), you can assume that your custom widget might end up getting placed on various canvas sizes. Based on the Agent's screen size and configuration by the Administrator, it could even end up with a different aspect ratio. Please keep that in mind while testing your responsive widget's behavior.
+
+# Development
+
+### Prerequisites
+
+Before starting, make sure you have yarn installed to be able to run the scripts in the `package.json` and are running node version `14.18.2` (this can be checked in your terminal with `node -v`).
+
+## Getting Started
+
+To run your widget on `localhost`, please navigate to widget's root directory in Terminal (Command line tool) and run the following commands (Assuming you have [`yarn`](https://classic.yarnpkg.com/en/docs/install/#mac-stable) installed globally on your machine):
+
+1. Clone this repo.
+2. Navigate to the **webex-contact-center-widget-starter/angular** folder.
+3. Run `yarn` or `yarn install` from the root of the repo.
+4. Run `ng serve` or `yarn start` for a development server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
```

Comment:
```ng serve``` will only work if they have angular cli installed

Reply:
I'll add angular cli to the prerequisites